sycl/test/usm/mixed_queue.cpp

```diff
@@ -0,0 +1,103 @@
+// RUN: %clangxx -fsycl %s -o %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: %GPU_RUN_PLACEHOLDER %t1.out
+
+//==-------------- mixed_queue.cpp - Mixed Memory test ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+class foo;
+int main() {
+  int *darray = nullptr;
+  int *sarray = nullptr;
+  int *harray = nullptr;
```

Comment:
It seems we don't need three pointers at the same time in this test.
Only one is required.

Reply:
Good call.